app/views/dashboard/index.html.erb

```diff
@@ -35,15 +35,15 @@
   <div class="govuk-grid-column-one-third">
     <div class="wrapper">
       <h1 class="govuk-heading-s">
-        <%= link_to "My caseload", my_caseload_path(@pom.staff_id), class:"govuk-link" %>
+        <%= link_to "My caseload", my_caseload_path(@user.staff_id), class:"govuk-link" %>
```

Comment:
The path doesn't need to take the staff_id here, it isn't used in the controller.